src/sentry/api/endpoints/organization_details.py

```diff
@@ -516,6 +521,35 @@ def put(self, request, organization):
                     data=changed_data,
                 )
 
+                # Temporarily writing org-level apdex changes to ProjectTransactionThreshold
+                # for orgs who don't have the feature enabled so that when this
+                # feature is GA'ed it captures the orgs current apdex threshold.
+                if "apdexThreshold" in changed_data and not features.has(
```

Comment:
```suggestion
                if serializer.validated_data.get("apdexThreshold") is not None and not features.has(
```

Could read from validated_data here as well.